src/components/shared/Slider/Slider.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export const Slider = ({ min, max, step, defaultValue, handleChange }) => {
+  // TODO: add styles
+  return (
+    <input
+      id="slider"
```

Comment:
What will happen if you had more than one slider in the same page? this id would be an issue... 🤔

Reply:
Good catch! I think that the `id` should be removed, if I am not wrong, it is not necessary, right?